CHANGELOG.md

```diff
@@ -9,6 +29,10 @@
 - Remove unused $regenerate\_ca parameter [\#268](https://github.com/theforeman/puppet-certs/pull/268) ([ekohl](https://github.com/ekohl))
 - /etc/pki/pulp perms and group owner [\#266](https://github.com/theforeman/puppet-certs/pull/266) ([wbclark](https://github.com/wbclark))
 
+**Fixed bugs:**
+
+- Fixes [\#25683](https://projects.theforeman.org/issues/25683) - Do not run postun script if upgrading [\#267](https://github.com/theforeman/puppet-certs/pull/267) ([jlsherrill](https://github.com/jlsherrill))
```

Comment:
This is a limitation of the changelog generator: it assumes linear history. To work around this, add things to https://github.com/theforeman/puppet-certs/blob/master/HISTORY.md to force it to behave.